### PR TITLE
cog example

### PR DESCRIPTION
**Pull request by yonipy**
## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: examples/cogs.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import voltage 
+import Cog
```
Comment (RGBCube) [inline examples/cogs.py:2]: Wrong

Comment (ysIgnacio) [inline examples/cogs.py:2]: use `from voltage.ext.commands import Cog`

**Review by ysIgnacio — CHANGES_REQUESTED** [review]

File: examples/cogs.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+def setup(client) -> Cog:
```
Comment (ysIgnacio) [inline examples/cogs.py:4]: client is not defined(?)

**Comment by ysIgnacio** [comment]
why close

**Pull request closed without merging**